File: 240728/Next Level/next-level.c
#include <stdio.h>
#include <string.h>

typedef struct user {
    char id[10];
    int lv;
} user;

int main() {
    // 여기에 코드를 작성해주세요.
    user user1;
    user user2;

    strcpy(user1.id, "codetree");
    user1.lv = 10;
    printf("user %s lv %d\n", user1.id, user1.lv);

    scanf("%s %d", user2.id, &user2.lv);
    printf("user %s lv %d", user2.id, user2.lv);

    return 0;
}

File: 240728/Next Level/next-level.py
class user:
    def __init__(self, user_id, user_lv):
        self.user_id = user_id
        self.user_lv = user_lv

user1 = user("codetree", 10)
print(f"user {user1.user_id} lv {user1.user_lv}")

user2_id, user2_lv = tuple(input().split())
user2 = user(user2_id, user2_lv)
print(f"user {user2.user_id} lv {user2.user_lv}")